fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/Checklist/Library_new.xml

```diff
@@ -0,0 +1,2610 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What is the reason you made a Library_new.xml?
And would a more specific name make sense? (e.g. Libraray_A32NX.xml or Library_flybywire.xml)

Reply:
The file currently exists within the a32nx package so adding a flybywire or a32nx suffix doesn't seem appropriate here. Additionally, making it generic (for the a380) may not be feasible, however, that depends on how much the variable names differ and how much the procedures differ. It's something that would probably have to be evaluated when it's closer to completion.

As for why it's a separate file, it's mostly because I ended up making large amounts of structural changes as I was going through it and it made it easier for me to modify what I had implemented and what was still, effectively, a stub. I was also considering splitting it into more files based on the location of the instrument being referred to, however, that doesn't work for all items so I left it in one file.

However, given how large the changes are the diff may look weird if it's added back into the same file (even as a reset). If you want I can go merge them back together and see how it looks.

Reply:
I just think we should avoid *_new* in filenames - "new" is always relative and quickly is "not new" any more 😀 
I'll leave it to you but maybe you can think of something more neutral/lasting.